src/raven.js

```diff
@@ -1779,6 +1780,13 @@ function objectMerge(obj1, obj2) {
     return obj1;
 }
 
+function objectFrozen(obj) {
```

Comment:
@HazAT – would you mind adding a comment about why we do this for react native, and why it's okay if `Object.isFrozen` isn't called in environments where it isn't supported (maybe link to this PR?). Last thing and then I'll merge.